java-checks/src/main/java/org/sonar/java/checks/UselessImportCheck.java

```diff
@@ -70,11 +71,13 @@
   public void scanFile(JavaFileScannerContext context) {
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) The Cyclomatic Complexity of this method "scanFile" is 11 which is greater than 10 authorized. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://nemo.sonarqube.org/coding_rules#rule_key=squid%3AMethodCyclomaticComplexity)